### ISSUE TITLE
inconsistency in parameter values

### ISSUE THREAD

**Issue by SweBarre** [HewlettPackard/POSH-HPOneView#244]
When using Get-HPOVServerProfileTemplate -name "Template Name" the returning object `$_.firmware.firmwareInstallType` has one of the following values if` $_.firmware.manageFirmware` is set to true: **FirmwareAndOSDrivers, FirmwareOnly, FirmwareOnlyOfflineMode**

When using New-HPOVServerProfileTemplate then -FirmwareMode only takes on of the following parameter **FirmwareAndSoftware, FirmwareOnly, FirmwareOffline**

I think that they should be the same, as it is today it adds a lot of extra coding when exporting, reconfigure and importing new profile templates that are based on existing templates.

### Expected Behavior
I expected that the returning values from Get-HPOVServerProfileTemplate and the property values needed by New-HPOVServerProfileTemplate would mach so that values returned from Get-HPOVServerProfileTemplate could be reused when by New-HPOVServerProfileTemplate

### Actual Behavior
The parameter values from Get-HPOVServerProfileTemplate and the New-HPOVServerProfileTemplate  doesn't match

### Steps to reproduce
* Create a Server Profile Template in OneView and and name it TestTemplate associate it to an Enclosure Group and Server Hardware type. In Firmware, select a firmware baseline and choose any installation method.

Run the following script
```
`
Import-Module HPOneView.300
$Appliance = Read-Host 'ApplianceName'
$Username  = Read-Host 'Username'
$Password  = Read-Host 'Password' -AsSecureString
$ApplianceConnection = Connect-HPOVMgmt -Hostname $Appliance -Username $Username -Password $Password
$MyTemplate = Get-HPOVServerProfileTemplate -Name "TestTemplate"

$params = @{
	Name               = "Copy of TestTemplate from POSH";
	ServerHardwareType = Get-HPOVServerHardwareType | where {$_.uri -eq $MyTemplate.serverHardwareTypeUri};
	EnclosureGroup     = Get-HPOVEnclosureGroup | where {$_.uri -eq $MyTemplate.enclosureGroupUri};
    Connections        = $MyTemplate.connections
    ManageBoot         = $MyTemplate.boot.manageBoot;
	Firmware           = $MyTemplate.firmware.manageFirmware;
	Baseline           = Get-HPOVBaseline | where {$_.uri -eq $MyTemplate.firmware.firmwareBaselineUri};
	FirmwareMode       = $MyTemplate.firmware.firmwareInstallType;
	BootMode           = $MyTemplate.bootMode.mode;
	PxeBootPolicy      = $MyTemplate.bootMode.pxeBootPolicy;
	BootOrder          = $MyTemplate.boot.order; 
	LocalStorage       = $true;
	StorageController  = $MyTemplate.localStorage.controllers;
	HideUnusedFlexnics = $MyTemplate.hideUnusedFlexNics;
}        

New-HPOVServerProfileTemplate @params | Wait-HPOVTaskComplete
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):
```
PowerShellLibraryVersion : 3.0.1221.2746
LibraryPath              : C:\Program Files\WindowsPowerShell\Modules\hponeview.300
```

HPE OneView Appliance Version:  
` Firmware 3.00.05-0271823`

Output from `$PSVersionTable` on your Windows Host:
```
Name                           Value                                                                                                 
----                           -----                                                                                                 
PSVersion                      4.0                                                                                                   
WSManStackVersion              3.0                                                                                                   
SerializationVersion           1.1.0.1                                                                                               
CLRVersion                     4.0.30319.42000                                                                                       
BuildVersion                   6.3.9600.17400                                                                                        
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0}                                                                                  
PSRemotingProtocolVersion      2.2  
```

**Comment by ChrisLynchHPE** [comment]
That's fair.  I'll maintain both sets of value so as to not break existing scripts.

**Comment by ChrisLynchHPE** [comment]
This is addressed in [Release 3.0.1264.2772](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1264.2772).

**Comment by ChrisLynchHPE** [comment]
Closing issue due to no further activity.  Feel free to re-open the issue if it still remains.
